src/matrix-utils.ts

```diff
@@ -161,16 +180,16 @@ export async function createRoom(client, name, isPtt = false) {
   console.log({ isPtt });
```

Comment:
While you're here, this should probably be an actual log message rather than a raw object